corehq/apps/users/templates/users/mobile_workers.html

```diff
@@ -0,0 +1,502 @@
+                            </a>
+                        {% endif %}
+                        {% if can_add_extra_users %}
+                            <a class="btn btn-default" href="{% url "user_fields_view" domain %}" id="btn-edit_user_fields">
+                                {% trans "Edit User Fields" %}
+                            </a>
+                        {% endif %}
+                    </div>
+                </div>
+            </div>
+            <div class="modal fade" id="newMobileWorkerModal">
+                <div class="modal-dialog">
+                    <form id="newMobileWorkerForm"
+                          name="mobileWorkerForm"
+                          novalidate
+                          ng-submit="submitNewMobileWorker()">
+                        <div class="modal-content">
+                            <div class="modal-header">
+                                <button type="button" class="close" data-dismiss="modal">
+                                    <span aria-hidden="true">&times;</span>
+                                    <span class="sr-only">{% trans 'Close' %}</span>
+                                </button>
+                                <h3 class="modal-title">{% trans "Create New Mobile Worker" %}</h3>
+                            </div>
+                            <div class="modal-body">
+                                <div class="form-horizontal">
+                                    {% crispy new_mobile_worker_form %}
+                                    {% if custom_fields_form.fields %}
+                                        {% crispy custom_fields_form %}
+                                    {% endif %}
+                                </div>
+                            </div>
+                            <div class="modal-footer">
+                                <button type="button" class="btn btn-default" data-dismiss="modal">{% trans 'Cancel' %}</button>
+                                <button type="submit"
+                                        ng-disabled="mobileWorkerForm.$invalid || usernameAvailabilityStatus !== 'available'"
+                                        class="btn btn-primary">
+                                    {% trans "Create" %}
+                                </button>
+                            </div>
+                        </div>
+                    </form>
+                </div>
+            </div>
+            {% angularjs %}
+
+            <div class="panel panel-info"
+                 ng-show="workers.length > 0"
+                 ng-cloak>
+                <div class="panel-heading">
+                    <h3 class="panel-title">
+                        {% trans 'New Mobile Workers' %}
+                    </h3>
+                </div>
+                <div class="panel-body">
+                    <table class="table table-striped">
+                        <thead workertable-header></thead>
```

Comment:
this is `workertable-header`, but the directive is `workertableHeader`, is that just an angular translation thing?